src/App.vue

```diff
@@ -70,11 +70,15 @@ export default {
   data() {
     return {
       nowTime: new Date(),
+      disableNightAutoToggle: false,
     };
   },
   created() {
     setInterval(() => {
       this.nowTime = new Date();
+      if (!this.disableNightAutoToggle) {
+        this.setNightMode(this.checkIsNightMode());
+      }
     }, 1000);
```

Comment:
I don't see a value in checking it here every second like this. We can make the check the first time the app is created and set the value there. If the user changes their browser dark mode setting, they will see this change applied in the first new window they open, not on the existing open windows, and that's fine in my opinion.

```suggestion
    this.setNightMode(this.checkIsNightMode());
    setInterval(() => {
      this.nowTime = new Date();
    }, 1000);
```